src/Adapter/Meta/ShopUrlDataConfiguration.php

```diff
@@ -107,6 +107,6 @@ public function validateConfiguration(array $configuration)
         return isset(
             $configuration['domain'],
             $configuration['domain_ssl']
-        );
+        ) && preg_match('#^(?:[~\-_\/&\.]|\w|%\d+|\s)+$#', $configuration['physical_uri']);
```

Comment:
What about `Validate::isUrl()` method, which seems easier to read here?

Reply:
It's not an url, it's an uri. Also because
`http://google.fr/"onmouseover="alert(123)"` is a valid url :)

Reply:
What do you think about creating a dedicating method or, at least, adding a comment on the expected check?

Reply:
Great idea! Done!